docs/source/en/perf_infer_gpu_one.md

```diff
@@ -17,7 +17,9 @@ rendered properly in your Markdown viewer.
 
 In addition to this guide, relevant information can be found as well in [the guide for training on a single GPU](perf_train_gpu_one) and [the guide for inference on CPUs](perf_infer_cpu).
 
-## Better Transformer: PyTorch-native transformer fastpath
+## Better Transformer: PyTorch-native transformer fastpath that uses Flash Attention
```

Comment:
Add the "PyTorch native transformer fastpath ... " under the header to give a brief explanation of what BetterTransforer is

```suggestion
## BetterTransformer
```